### PR TITLE
Item Types: add new collection type

### PR DESCRIPTION
**Pull request by danielbeardsley**
Meant to store a picture associated with item types.

Connect iFixit/ifixit#49086

### CODE REVIEW

File: backend/src/api/item-type/content-types/item-type/schema.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+   "options": {
+      "draftAndPublish": true
+   },
+   "pluginOptions": {},
+   "attributes": {
+      "akeneo_code": {
+         "type": "string",
+         "required": true,
+         "unique": true,
+         "regex": "[a-z][a-z0-9]*"
```
Comment (sterlinghirsh) [inline backend/src/api/item-type/content-types/item-type/schema.json:18]: Not sure if this is getting used, but we need to support underscores.

**Comment by sterlinghirsh** [comment]
CR :zap: dev_block :zap: on making sure underscores will be allowed contrary to the regex in the code.

File: backend/src/api/item-type/content-types/item-type/schema.json (hunk 1 above)
Comment (danielbeardsley) [reply backend/src/api/item-type/content-types/item-type/schema.json]: Fixed.

**Comment by danielbeardsley** [comment]
un_dev_block :+1:

**Review by masonmcelvain — CHANGES_REQUESTED** [review]
We need to add `api::item-type.item-type` to `backend/src/bootstrap/permissions/config.json`, otherwise we'll Next.js will get a 401 when trying to fetch these resources

**Comment by danielbeardsley** [comment]
Fixed! Thanks for the tip!

**Review by masonmcelvain — APPROVED** [review]
CR 👏🏻 Nice!

**Comment by danielbeardsley** [comment]
QA :+1: 

Strapi on govinor has the new type:
image

**Pull request merged**